project3/index1.html

```diff
@@ -25,254 +25,340 @@ <h2>With great power there must also come-great responsibility</h2>
-<label for="prefer not to answer">
-            <input name="gender" id="prefer not to answer" type="radio">
-            Prefer not to answer</label>
-    <label for="prefer to self describe">
-            <input name="gender" id="prefer to self describe" type="radio">
-        Prefer to self describe</label>
-            <textarea name="" id="prefer to self describe"></textarea>
-        </fieldset>
-</div>
+            </label>
+            <div class="category">
+                <fieldset>
+                    <legend>Category</legend>
+                    <div class="description">
+                        <p>Please select as many as apply to your origin and how you gained your powers.</p>
+                    </div>
+                    <lable for="alien">
+                        <input id="alien" type="checkbox">
+                        Alien</label>
+                        <label for="cyborg">
+                            <input id="cyborg" type="checkbox">
+                            Cyborg</label>
+                        <label for="experiment">
+                            <input id="experiment" type="checkbox">
+                            Experiment</label>
+                        <label for="human">
+                            <input id="human" type="checkbox">
+                            Human</label>
+                        <label for="inhuman">
+                            <input id="inhuman" type="checkbox">
+                            Inhuman</label>
+                        <label for="mutant">
+                            <input id="mutant" type="checkbox">
+                            Mutant</label>
+                </fieldset>
+            </div>
+            <div class="gender">
+                <fieldset>
+                    <legend>Gender</legend>
+                    <label for="female">
+                        <input name="gender" id="female" type="radio">
+                        Female</label>
+                    <label for="male">
+                        <input name="gender" id="male" type="radio">
+                        Male</label>
+                    <label for="nonbinary">
+                        <input name="gender" id="nonbinary" type="radio">
+                        Nonbinary</label>
+                    <label for="genderqueer/genderfluid">
+                        <input name="gender" id="genderqueer/genderfluid" type="radio">
+                        Genderqueer/Genderfluid</label>
+                    <label for="prefer not to answer">
+                        <input name="gender" id="prefer not to answer" type="radio">
+                        Prefer not to answer</label>
+                    <label for="prefer to self describe">
+                        <input name="gender" id="prefer to self describe" type="radio">
+                        Prefer to self describe</label>
+                    <textarea name="" id="prefer to self describe"></textarea>
```

Comment:
This `textarea` can't have the same `id` as the `input` above; `id`s must be unique.